sample/src/main/AndroidManifest.xml

```diff
@@ -8,9 +8,12 @@
 
     <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE"/>
     <uses-permission android:name="android.permission.INTERNET"/>
-    <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE"/>
     <uses-permission android:name="android.permission.ACCESS_FINE_LOCATION" />
 
+    <!-- not needed as of api 23 -->
+    <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE"/>
```

Comment:
according to the docs we could do this

```
<uses-permission
        android:name="android.permission.WRITE_EXTERNAL_STORAGE"
        android:maxSdkVersion="22" />
<uses-permission
        android:name="android.permission.READ_EXTERNAL_STORAGE"
        android:maxSdkVersion="22" />
```

from:
https://developers.google.com/maps/documentation/android-api/config

it looks like a future version will drop the support completely. 

I'm currently inclined to leave it as is as most people have other reasons for needing the permissions